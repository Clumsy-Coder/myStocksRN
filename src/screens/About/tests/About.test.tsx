import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';

import AboutScreen from '@screens/About';

describe('<AboutScreen />', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<AboutScreen />);

    expect(wrapper).toMatchSnapshot();
  });
});
