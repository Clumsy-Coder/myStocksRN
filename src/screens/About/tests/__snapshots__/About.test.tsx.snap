// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutScreen /> Renders correctly 1`] = `
<Styled(Container)
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
    }
  }
>
  <Styled(Content)
    style={
      Object {
        "paddingTop": 30,
      }
    }
  >
    <Styled(H1)>
      Build version: 
    </Styled(H1)>
  </Styled(Content)>
</Styled(Container)>
`;
